Learn_go/网络通信/learn02-web/server/main.go

```
@@ -0,0 +1,26 @@
+	"fmt"
+	"net/http"
+)
+
+// 服务器端
+func main() {
+	fmt.Printf("准备开启服务器…………")
+	http.HandleFunc("/hello", hello)
+	handler1 := handler1{}
+
+	http.Handle("/helloA", &handler1)
+	http.ListenAndServe(":8081", nil)
```

golangci-lint (1.64.8)

Error return value of `http.ListenAndServe` is not checked

(errcheck)